accurest-converters/src/test/groovy/io/codearte/accurest/wiremock/DslToWireMockClientConverterSpec.groovy

```diff
@@ -28,6 +29,30 @@ class DslToWireMockClientConverterSpec extends Specification {
 ''', json, false)
 	}
 
+	@Issue("196")
+	def "should creation of delayed stub responses be possible"() {
+		given:
+			def converter = new DslToWireMockClientConverter()
+		and:
+			String dslBody = """
+				io.codearte.accurest.dsl.GroovyDsl.make {
+					request {
+						method "GET"
+						url "/test"
+					}
+					response {
+						status 200
+						fixedDelayMilliseconds 1000
+					}
+			}
+"""
+		when:
+			String json = converter.convertContent(dslBody)
+		then:
+			JSONAssert.assertEquals('''
+{"request":{"method":"GET","url":"/test"},"response":{"status":200, "fixedDelayMilliseconds":1000}}
```

Comment:
I will consider checking only `response.fixedDelayMilliseconds` value instead of full json. That will show exactly the subject of this test.